src/components/contestants.jsx

```diff
@@ -0,0 +1,245 @@
+        "slug": "multimedia"
+       },
+       {
+        "title": "Velocity Reporter",
+        "authors": ["mihaela costache", "bogdan nechifor"],
+        "county": "brăila",
+        "comments": 5,
+        "category": "web",
+        "slug": "web"
+       }],
+     showGrid: isMobile ? false : true,
+     showTable: isMobile ? true : false,
+     showCategory: {
+        "educational": true,
+        "utility": true,
+        "web": true,
+        "media": true,
+        "robots": true
+     }
+   };
+
+  },
+
+  showGrid() {
+    this.setState({
+      showTable: false,
+      showGrid: true
+    });
+  },
+
+  showTable() {
+    this.setState({
+      showTable: true,
+      showGrid: false
+    });
+  },
+
+  toggleCategory(category) {
+    var newStates = this.state.showCategory;
+    newStates[category] = this.state.showCategory[category] ? false : true;
+
+    this.setState({ showCategory: newStates });
+  },
+
+  render() {
+    var gridClassName = "icon hidden-xs " + (this.state.showGrid ? "" : "inactive");
+    var tableClassName = "icon hidden-xs " + (this.state.showTable ? "" : "inactive");
+    var categories = this.state.showCategory;
+
+    return <div className="contestants">
+        <div className="blue-section-wrapper">
+          <Grid className="blue-section">
+            <Header isLoggedIn={this.props.isLoggedIn} />
+            <Row className="xsmall-spacing" />
+            <Row>
+              <Col md={12}>
+                <h1>Participanți InfoEducație Ediția 2015</h1>
+              </Col>
+            </Row>
+            <Row>
+              <Col md={12}>
+                <h2>Concurs Național de Informatică</h2>
+              </Col>
+            </Row>
+            <Row className="big-spacing" />
+            <Row>
+              <Col md={6} mdOffset={3} className="middle-align hidden-xs hidden-ms">
+                <Row className="stats">
+                  <Col md={4} xs={2} className="right-border">
+                      <p className="description">Participanți</p>
+                      <p className="value">75</p>
+                  </Col>
+                  <Col md={4} xs={2} className="right-border">
+                      <p className="description">Proiecte</p>
+                      <p className="value">39</p>
+                  </Col>
+                  <Col md={4} xs={2} >
+                      <p className="description">Județe</p>
+                      <p className="value">17</p>
+                  </Col>
+                </Row>
+              </Col>
+            </Row>
+          </Grid>
+        </div>
+        <Row className="big-spacing" />
+        <div className="white-section-wrapper">
+          <Grid>
+            <Row className="white-section">
+              <Col mdOffset={2} md={1}>
+                <div onClick={this.toggleCategory.bind(this, "educational")}
+                     className={"round-icon " + (this.state.showCategory.educational ? "" : "inactive") }>
```

Comment:
Ma gandeam la un `map` prin categori si sa spargem div-urile alea in componente separate